tests/decay_both_directions/__main__.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
In the previous commit you were renaming this files. Are there 2 copies of this test now?

Reply:
I renamed the folder only.